kotlinsampleapp/src/main/java/in/co/kotlinsampleapp/CustomPushMessageListener.kt

```diff
@@ -0,0 +1,57 @@
+package `in`.co.kotlinsampleapp
```

Comment:
Follow the same package structure in both sample apps, move the callbacks to callbacks package

Reply:
Done